src/pages/Transaction/Tabs/Components/TransactionFunction.tsx

```diff
@@ -56,7 +56,7 @@ export default function TransactionFunction({
     functionFullStr === "0x1::aptos_account::transfer"
   ) {
     return (
-      <InternalLink
+      <Link
```

Comment:
wanna have underline as none for coin transfer?

Reply:
done